lib/internal/errors.js

```diff
@@ -80,4 +80,46 @@ module.exports = exports = {
 //
 // Note: Please try to keep these in alphabetical order
 E('ERR_ASSERTION', (msg) => msg);
+E('ERR_CHANNEL_CLOSED', 'channel closed');
+E('ERR_INVALID_ARG_TYPE', invalidArgType);
+E('ERR_INVALID_HANDLE_TYPE', 'This handle type cannot be sent');
+E('ERR_INVALID_OPT_VALUE',
+  (name, value) => {
+    return `The value "${String(value)}" is invalid for option "${name}"`;
+  });
+E('ERR_INVALID_SYNC_FORK_INPUT',
+  (value) => {
+    return 'Asynchronous forks do not support Buffer, Uint8Array or string' +
+           `input: ${value}`;
+  });
+E('ERR_IPC_DISCONNECTED', 'IPC channel is already disconnected');
+E('ERR_IPC_ONE_PIPE', 'Child process can have only one IPC pipe');
+E('ERR_IPC_SYNC_FORK', 'IPC cannot be used with synchronous forks');
+E('ERR_REQUIRED_ARG',
+  (name) => {
+    return `The "${name}" argument is required and cannot be undefined`;
+  });
+E('ERR_UNKNOWN_SIGNAL', (signal) => `Unknown signal: ${signal}`);
 // Add new errors from here...
+
+function invalidArgType(name, expected, actual) {
```

Comment:
If I read it correctly, this would print `The "options" argument must be type Object`. It looks better to me as `The "options" argument must be of type Object`